drivers/entropy/entropy_stm32.c

```diff
@@ -143,6 +145,7 @@ static int entropy_stm32_resume(const struct device *dev)
 	res = clock_control_on(dev_data->clock,
 			(clock_control_subsys_t)&dev_cfg->pclken[0]);
 	LL_RNG_Enable(rng);
+	LL_RNG_EnableIT(rng);
```

Comment:
Looks it solves the on going issues and hence a welcome addition, but is there a proper explanation ?

Reply:
I did some testing and it looks like the IE bit in CR register is sometimes cleared when the device clock is disabled. It looks like it happens only on H7 chip. Unfortunately, I haven't fine any confirmation of such behavior in RM.